src/settings-ui/Settings.UI/ViewModels/ShellViewModel.cs

```diff
@@ -38,21 +38,6 @@ public bool IsBackEnabled
             set { Set(ref isBackEnabled, value); }
         }
 
-        public bool IsVideoConferenceBuild
-        {
-            get
-            {
-                var mfHandle = NativeMethods.LoadLibrary("mf.dll");
```

Comment:
We need to check if mf.dll is available, as as far as I know it is needed for VCM.
It is not pre installed in the Windows N edition.

Reply:
Where and for what is this needed? `IsVideoConferenceBuild` is _only_ referred to from the ShellPage > NavigationView

Reply:
Runner checks also for thia dll https://github.com/microsoft/PowerToys/blob/main/src/runner/main.cpp

MF is used all over the place in the VCM code

Reply:
Hm, I see, I wasn't aware of that... Even though
> Users of N editions can make websites and software work correctly by installing the [Media Feature Pack](https://support.microsoft.com/en-us/topic/media-feature-pack-for-n-editions-of-windows-10-version-1607-b657cb70-33e7-1f11-7119-3b4b50be4e89#bkmk_download) for N editions of Windows 10 (KB3133719).

Does that mean we can't change this? Or should we leave responsibility for this ability at the user?

Reply:
@yuyoyuppe can help here

Reply:
> Hm, I see, I wasn't aware of that... Even though
> > Users of N editions can make websites and software work correctly by installing the [Media Feature Pack](https://support.microsoft.com/en-us/topic/media-feature-pack-for-n-editions-of-windows-10-version-1607-b657cb70-33e7-1f11-7119-3b4b50be4e89#bkmk_download) for N editions of Windows 10 (KB3133719).
> 
> Does that mean we can't change this? Or should we leave responsibility for this ability at the user?

I think a warning, when the Pack isn't installed on the Settings page would be more appropriate

Reply:
@Aaron-Junker is correct, we need this

Reply:
Is this a bad time to confess I thought this all was mostly a left-over from the old "experimental" branch?
I'll revert this.